### DISCUSSION TITLE
[Crostini] Use and USB audio device for making music

### DISCUSSION THREAD

**Discussion by MeneerJansen** [dnschneid/crouton#4711] [Q&A] [answered]
I use my old Chromebook and Crouton for making music with a DAW (digital audio workstation). One needs an external audio device for that. It works in Crouton, but will it in Crostini?

**Comment by DennisLfromGA** [comment, marked as answer]
@MeneerJansen,

Good question, I hope the Crostini devs will eventually allow better access to the hardware ports but we'll just have to wait and see. 

The '[Chromium OS Docs - Running Custom Containers Under Chrome OS](https://chromium.googlesource.com/chromiumos/docs/+/HEAD/containers_and_vms.md#missing-features)' document states:
> Missing Features
There are still many features we're working on fleshing out.

> Peripheral access ([USB](https://crbug.com/831850), Bluetooth, [Camera](https://crbug.com/907701)/etc...).
[IMEs](https://crbug.com/826614).
There are more things we‘re thinking about, but we’re being very careful/cautious in rolling out features as we want to make sure we aren't compromising overall system security in the process. The (large) FAQ below should hopefully hit a lot of those topics.

This isn't really a crouton question so I doubt the crouton devs will address this. you might want to ask this question on the ['Chromium OS Discussion' forum](https://groups.google.com/a/chromium.org/g/chromium-os-discuss), sometimes the CrOS devs will drop by and address questions there.

*Hope this helps,*
-**DennisLfromGA**
